contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -1184,6 +1184,15 @@ class tsqlCommonMutator : public TSqlParserBaseListener
 			rewritten_query_fragment.emplace(std::make_pair(linked_srv->start->getStartIndex(), std::make_pair(linked_srv_name, str)));
 		}	
 	}
+
+	void exitSTRING_AGG(TSqlParser::STRING_AGGContext *ctx) override
```

Comment:
Nit: Please add comment regarding why this change is needed

Reply:
Added

Reply:
Upon careful investigation find out that, we do not need to rewrite in `exitSTRING_AGG` function as `exitSTRING_AGG` function will only be called if `STRING_AGG` function is using `WITHIN GROUP` and since `WITHIN GROUP` is supported from Babelfish v16.4 onwards, we do not need to rewrite the `STRING_AGG` here.